pkg/controllers/blog.go

```diff
@@ -7,37 +7,42 @@ import (
 )
 
 
-func (c *Controller) GetBlogHandler(w http.ResponseWriter, r *http.Request) {
-	userID := context.Get(r, "id").(int)
-	results := c.BlogStore.GetBlog(userID)
+func (c *Controller) GetBlogHandler() http.HandlerFunc{
+	return func (w http.ResponseWriter, r *http.Request){
+		userID := context.Get(r, "id").(int)
+		results := c.BlogStore.GetBlog(userID)
 
-
-	tmpl, _ := template.ParseFiles("./web/templates/blog.html")
-	tmpl.Execute(w, results)
+		tmpl, _ := template.ParseFiles("./web/templates/blog.html")
+		tmpl.Execute(w, results)
```

Comment:
Please use functions in package "view"

Reply:
No way. This was made for personal convenience. You play your chessboard in your part.

Reply:
What do you mean by personal convenience? All members of the team use 'view' package for generating html-templates. GenerateHtml() func was written for that, for convenience. Let's use similar approaches for routine actions. If you have another, more rational option, please suggest it.